src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -156,6 +180,7 @@
      * @param w  The world the colony exists in.
      * @param c  The center of the colony (location of Town Hall).
      */
+    @SuppressWarnings("squid:S2637")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Suppressing the 'squid:S2637' warning is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)